internal/controller/controllers/hypershiftserviceconfig_controller_test.go

```diff
@@ -0,0 +1,83 @@
+package controllers
+
+import (
+	"context"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	aiv1beta1 "github.com/openshift/assisted-service/api/v1beta1"
+	"github.com/openshift/assisted-service/internal/common"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	fakeclient "sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+const (
+	kubeconfigSecret = "secret"
```

Comment:
This feels a bit generic for a constant accessible to all the code in this package.
I'd prefer you put this in the `Describe` scope so the non-test controller code doesn't see, and maybe by mistake, use it.